src/app/contextBroker/contextBroker.cpp

```diff
@@ -1422,7 +1450,55 @@ static SemRequestType policyGet(std::string mutexPolicy)
   return SemReadWriteOp;
 }
 
+/* ****************************************************************************
+*
+* notificationModeParse -
+*/
```

Comment:
Describe what is parsed here in the function header - makes it very easy to get what the function really does.

Personally I am not a friend of grouping varios CLI params info one like this.
What this function does is already implemented in the parseArgs lib, with default values, limit checks etc.